### ISSUE TITLE
Mandatory fields do not let you buy a ticket

### ISSUE THREAD

**Issue by cougil** [alfio-event/alf.io#561]
**Describe the bug**
It is not possible buy a ticket if you added few mandatory fields to fill

**To Reproduce**
1. Create mandatory fields
2. Go to the main site and select a ticket
3. Click on `I don't know yet who will attend the event`
4. Scroll down to the payment process
4. You won't be able to continue and pay your ticket :-(

**Expected behavior**
Continue the process if the user wants to pay and include the mandatory information later on

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
Alf.io v.1.17-SNAPSHOT

**Comment by syjer** [comment]
Hi @cougil , you are right, it's a bug that will be fixed .

As a work around you can uncheck:

![2018-12-10](https://user-images.githubusercontent.com/498146/49729532-0f6de180-fc76-11e8-9575-6b6b8299ff4f.png)

**Comment by cougil** [comment]
Hi @syjer 
Thank you for your suggestion. We will enable this option in order to make the data mandatory... but as you can imagine it is not the best way to solve this :-/
Do you have an idea about how difficult will be to fix it? I mean, I remember it was something already detected in previous versions, wasn't it?
Best,

**Comment by cougil** [comment]
Hi @cbellone !
I've tested and it seems that it is working right now! Thank you very much! ;-)
About the issue I was thinking it was already shared. In fact, I thought I did raised it time ago, ... but I didn't ;-) I think it was [this one](https://github.com/alfio-event/alf.io/issues/399) the one I was referring, and it is not exactly the same thing that we are discussing here. 
So thank you once again ;-)
